### PR TITLE
customize annotation for scheduler configmap

### PR DESCRIPTION
**Pull request by hihihuhu**
### What problem does this PR solve?
using spinnaker to deploy the manifest generated by `helm template` requires additional annotation for the configmap, so that it won't and add version to the name

### What is changed and how does it work?
pass through the annotation to the config map

### Code changes

- [ ] Has Go code change
- [ ] Has CI related scripts change

### Tests

- [ ] Unit test 
- [ ] E2E test 
- [x] Manual test 
- [ ] No code 

`helm template` with the value set, unset and comment out

### Side effects

- [ ] Breaking backward compatibility 
- [ ] Other side effects: 

### Related changes

- [ ] Need to cherry-pick to the release branch 
- [ ] Need to update the documentation 

### Release Notes

Please refer to [Release Notes Language Style Guide](https://github.com/pingcap/tidb-operator/blob/master/docs/release-note-guide.md) before writing the release note.

```release-note
NONE
```

### CODE REVIEW

File: charts/tidb-operator/values.yaml [hunk 1]
```diff
@@ -163,6 +163,10 @@ scheduler:
   # Refer to https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
   podAnnotations: {}
 
+  # additional annotations for the configmap, mainly to prevent spinnaker versioning the cm
+  configmap:
+    annotations: {}
```
Comment (csuzhangxc) [inline charts/tidb-operator/values.yaml:168]: how about renaming it to something like `configmapAnnotations` just as `podAnnotations` did?

**Review by csuzhangxc — APPROVED** [review]

File: charts/tidb-operator/values.yaml (hunk 1 above)
Comment (hihihuhu) [reply charts/tidb-operator/values.yaml]: thanks for pointing out! fixed

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Pull request merged**